WordPress/wporg-two-factor
# New copy for TOTP + Notice reposition

Fixes #104 #105 #108 

This PR 
1. Updated the copy and combined the TOTP steps into a single step.
2. Moved the Error Notice to a new position.
3. Changed "cancel" to "clear", and moved the "previous" button on the left of the UI.

## Screencast

https://user-images.githubusercontent.com/18050944/232840205-c8dcb4df-4a5e-42b6-86dc-9a49ddf10408.mp4


## Testing Steps

[Figma](https://www.figma.com/file/vmHxwa8WhiyiP9poe2om9J/Inspector-Panels?node-id=14591-120142&t=SjIDl48SpGin2L1W-0)
1. Go to `Two-Factor Authentication`.
2. Check if the left button is with the text `Clear`
3. Check if the setup instruction copy is updated.
4. Try some random digits.
5. Error notice should appear in the middle with an animation effect after submitting.
6. Digits, red borders, and the error notice should disappear after clicking `Clear`.
7. Try the same steps in manual mode.
